.travis.yml

```diff
@@ -12,6 +12,7 @@ android:
   components:
     - platform-tools
     - tools
+    - tools
```

Comment:
Why we need second line with `- tools` ?

Reply:
from what i can remember, it ensures that it uses the most up-to-date sdk

Reply:
@TacoTheDank Actually, Travis ignores the duplicated lines.